test_common/harness/imageHelpers.cpp

```diff
@@ -801,6 +801,9 @@ void get_max_sizes(
     {
 
         size_t M = maximum_sizes[0];
+        size_t A = max_pixels;
+
+        M = static_cast<size_t>(fmin(A / M, M));
```

Comment:
This can cause M to be 0 because of the division.

Reply:
As `x0` can be 0 for `IMAGE1D_ARRAY` and `IMAGE2D`, and `x0` and `x1` can be 0 for `IMAGE2D_ARRAY` and `IMAGE3D`.
I'll add a `fmax` with `1` for this case that I'm modifying to make sure we never end up with `0`.